clients/web/src/dialog.js

```diff
@@ -74,7 +74,9 @@ function confirm(params) {
     }, params);
     $('#g-dialog-container').html(ConfirmDialogTemplate({
         params: params
-    })).girderModal(false);
+    })).girderModal(false).one('hidden.bs.modal', function () {
```

Comment:
To answer my own question, this is [a jquery function](http://api.jquery.com/one/), not a backbone function.